policy/camera.yaml

```diff
@@ -0,0 +1,188 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+#
+--- # auxTel
+#
+# There are nine ccds with names "S00".."S22" and IDs 0..8
+# Each has sixteen amplifiers with names like "00".."07" and "10".."17"
+
+name : "auxTel"
+plateScale : 13.55                      # plate scale, in arcseconds on sky/mm
+radialCoeffs : [0.0, 1.0, 0.0, 1.0]     # radial distortion coefficients (c_0 + c_1 r + c_2 r^2 + ...)
+
+#
+# A single amplifier ("segment" to the camera team)
+#
+AMP_ITL : &AMP_ITL
+    # trimmed
+    readCorner : LL
+
+    rawBBox                 : [[0,    0], [544,    2048]]  # total size of one amp's raw data
+
+  # name of section     : [[start,start], [width, height]] # start is zero-based, length=length of arr
+    rawDataBBox             : [[3,    0], [509,    2000]]  # data region in raw data
+    rawSerialPrescanBBox    : [[0,    0], [3,      2000]]  # serial prescan (often == extended)
+    rawSerialOverscanBBox   : [[512,  0], [32,     2000]]  # serial overscan
+    rawParallelPrescanBBox  : [[0,    1], [0,         0]]  # pixels digitised before first parallel transfer
+    rawParallelOverscanBBox : [[0, 2000], [509,      48]]  # parallel overscan
+
+    saturation : 262144                  # saturation level, DN XXX Should this be in electrons?
```

Comment:
I think this depends on whether we mean numerical saturation or full well.  I was assuming that it mean numerical saturation and that full well was slightly higher than numerical saturation.

Reply:
This should actually be used to mean full-well, and _not_ numerical saturation, and it's actually the other way around to what you might (do) think, and the reason is ~interesting: the full-well needs to be digitizable in order to apply cross-talk corrections correctly.

However, without knowing the actual full-well values, using the numerical saturation is a decent placeholder.

More importantly though, what would you like done with this line of code? Just leave as-is?